assets/src/edit-story/app/index.js

```diff
@@ -33,6 +34,7 @@ function App( { config } ) {
 						<HistoryProvider size={ 50 }>
 							<StoryProvider storyId={ storyId }>
 								<GlobalStyle />
+								<CropMoveableGlobalStyle />
```

Comment:
Should the existing `GlobalStyle` perhaps include all the global style? Not sure what's the best practice here. cc @barklund

Reply:
Yeah. Just let me know the best practice here.

Reply:
Pinging @barklund again in case that was missed.

Reply:
I think I prefer the separation here, but if that's not considered a good practice we can change it in a follow-up PR.

Reply:
This works for me. It's a bit of a special case as we're dealing with styling an external component, so this makes sense to me.